From c806632784655ea3281c46c8648d93e992bd9aa4 Mon Sep 17 00:00:00 2001
From: B00625 <B00625@freescale.com>
Date: Fri, 11 Jun 2010 16:17:42 +0800
Subject: [PATCH 05/43] MA-5839-2 added title for maddev_lp5 upgrade

Delete the SMS in ME memory each time SIM card is ready.
avoid ME memory full as it can only hold 23 SMSs

Signed-Off-By: Yinhaiting <b00625@freescale.com>
---
 reference-ril/reference-ril.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/reference-ril/reference-ril.c b/reference-ril/reference-ril.c
index c6546fb..c485655 100755
--- a/reference-ril/reference-ril.c
+++ b/reference-ril/reference-ril.c
@@ -381,6 +381,7 @@ static void onSIMReady()
      */
 at_send_command("AT+CNMI=1,2,2,1,1", NULL);
 at_send_command("AT+CPMS=\"ME\",\"ME\",\"ME\"",NULL);
+at_send_command("AT+CMGD=1,4",NULL);
 }
 
 static void requestRadioPower(void *data, size_t datalen, RIL_Token t)
-- 
1.8.0

